servers/quarkus-common/src/main/java/org/projectnessie/quarkus/providers/storage/PersistProvider.java

```diff
@@ -98,10 +99,14 @@ public Persist produceWithInitializedRepository(@Default Persist persist) {
     return persist;
   }
 
+  public void eagerPersistInitialization(
+      @Observes StartupEvent event, @NotObserved Persist persist) {
+    LOGGER.debug("Eager initialization of persist implementation '{}'", persist.name());
```

Comment:
Could you add a comment about using this to force `Persist` initialization?

Reply:
done